languages/go/generic/go_to_generic.ml

```diff
@@ -226,6 +226,26 @@ let top_func () =
             let st = G.OtherStmt (G.OS_Todo, [ G.T ty ]) |> G.s in
             G.F st)
   and expr_or_type v = either expr type_ v
+  (* used for translating call make/call new *)
+  and gen_new ty (l, args, r) t name =
+    let return (ty, args) =
+      G.New (fake t name, ty, G.empty_id_info (), (l, args, r))
+    in
+    let args = arguments args in
+    (* this translation (esp for make) depends on the first argument,
+     * where
+     *)
+    match ty with
+    (* the first arg is indeed a type *)
+    | ArgType ty -> return (type_ ty, args)
+    (* metavar *)
+    | Arg (Id v1) -> return (G.TyN (G.Id (v1, G.empty_id_info ())) |> G.t, args)
+    (* ... *)
+    | Arg (Ellipsis tok) ->
+        return (G.TyEllipsis tok |> G.t, G.Arg (G.Ellipsis tok |> G.e) :: args)
+    (* exp *)
+    | Arg exp -> return (G.TyExpr (expr exp) |> G.t, args)
+    | _ -> raise Impossible
```

Comment:
better to avoid those _ when we can and instead explicitely list the remaining cases, that way
if one day we add a new constructor in Arg, the compiler can tell us maybe we need to update this code.

Reply:
Actually we used to have a semgrep check for that, not sure why it's not triggering here.
Or maybe someone removed this check.